### PR TITLE
[RI2] Update CNTT/OPNFV to Anuket

### PR DESCRIPTION
**Pull request by rihabbanday**

### CODE REVIEW

**Review by CsatariGergely — APPROVED** [review]

**Review by walterkozlowski — APPROVED** [review]

**Review by ulikleber — APPROVED** [review]

File: doc/ref_impl/cntt-ri2/chapters/chapter01.md [hunk 1]
```diff
@@ -55,12 +55,13 @@ The scope of this document is as follows:
...
 
-- **OPNFV**
-  - The **Kuberef** project in OPNFV acts as the main integration point for RI2: Its purpose it to provide all necessary glue code and mechanisms for i) integrating and configuring selected upstream Kubernetes deployers and components into a deployable platform, ii) continuously deploy the integrated platform in OPNFV labs and iii) run the latest RC2 test suite against the deployed platform. The test results are meant to be fed back to RA2 and RC2 to support the refinement of architecture requirements and the selection of test cases for the RC2 test suite.
+- **Anuket Kuberef**
+  - The **Kuberef** project in Anuket acts as the main integration point for RI2: Its purpose it to provide all necessary glue code and mechanisms for i) integrating and configuring selected upstream Kubernetes deployers and components into a deployable platform, ii) continuously deploy the integrated platform in OPNFV labs and iii) run the latest RC2 test suite against the deployed platform. The test results are meant to be fed back to RA2 and RC2 to support the refinement of architecture requirements and the selection of test cases for the RC2 test suite.
```
Comment (jgu17) [inline doc/ref_impl/cntt-ri2/chapters/chapter01.md:62]: OPNFV labs >Anuket labs

**Review by jgu17 — CHANGES_REQUESTED** [review]
One nit: OPNFV labs->Anuket labs

File: doc/ref_impl/cntt-ri2/chapters/chapter01.md (hunk 1 above)
Comment (rihabbanday) [reply doc/ref_impl/cntt-ri2/chapters/chapter01.md:62]: Thanks for pointing out - I didn't change the labs part as I wasn't sure if Anuket Labs is the new name for former OPNFV labs - so far the lab related services like jira, gerrit, etc. still seem to be tied to OPNFV. Nevertheless, I'll rename it to Anuket Labs.

File: doc/ref_impl/cntt-ri2/chapters/chapter04.md [hunk 2]
```diff
@@ -15,7 +15,7 @@
...
 
-This chapter documents the steps to deploy Kubernetes based Reference Implementation (RI-2) according to RA-2. The entire deployment has been tested in OPNFV Labs as a part of the [OPNFV Kuberef project](https://wiki.opnfv.org/display/KUB/Kuberef). The Kuberef project stores all the code needed to deploy RI-2 and hence serves as a reference platform for CNF vendors to develop and test against. Currently, Kuberef only focuses on bare-metal RI2 deployments, but support for other uses cases (running on Packet, other infrastructure providers, etc.) will be added as the development progresses.
+This chapter documents the steps to deploy Kubernetes based Reference Implementation (RI-2) according to RA-2. The entire deployment has been tested in OPNFV Labs as a part of the [OPNFV Kuberef project](https://wiki.opnfv.org/display/KUB/Kuberef). The Kuberef project stores all the code needed to deploy RI-2 and hence serves as a reference platform for CNF vendors to develop and test against. Currently, Kuberef only focuses on bare-metal RI2 deployments, but support for other use cases (running on Packet, other infrastructure providers, etc.) will be added as the development progresses.
```
Comment (michaelspedersen) [inline doc/ref_impl/cntt-ri2/chapters/chapter04.md:18]: Nit: "Packet" -> "Equinix Metal"

Comment (michaelspedersen) [reply doc/ref_impl/cntt-ri2/chapters/chapter04.md:18]: Also, consistency across chapters for RA2/RI2/RC2 vs. RA-2/RI-2/RC-2 (could be done in a separate PR)

**Review by michaelspedersen — COMMENTED** [review]
Added a few notes, otherwise good

File: doc/ref_impl/cntt-ri2/chapters/chapter04.md (hunk 2 above)
Comment (rihabbanday) [reply doc/ref_impl/cntt-ri2/chapters/chapter04.md:18]: > Also, consistency across chapters for RA2/RI2/RC2 vs. RA-2/RI-2/RC-2 (could be done in a separate PR)

That's a good point - I've observed that these inconsistencies exist at various places throughout the CNTT documentation (not only RI2). I will try to check if there are any specific guidelines for this and fix it in another patch.

**Review by CsatariGergely — APPROVED** [review]

File: doc/ref_impl/cntt-ri2/chapters/chapter04.md (hunk 2 above)
Comment (CsatariGergely) [reply doc/ref_impl/cntt-ri2/chapters/chapter04.md]: There are no guidelines yet and I think we should start a wiki page to collect these.

**Review by michaelspedersen — APPROVED** [review]

File: doc/ref_impl/cntt-ri2/chapters/chapter04.md (hunk 2 above)
Comment (rihabbanday) [reply doc/ref_impl/cntt-ri2/chapters/chapter04.md]: > There are no guidelines yet and I think we should start a wiki page to collect these.

I was thinking of opening an issue to address this, but a wiki page could be a good starting point as well. I am not quite sure where within the Anuket wiki the page should exist - Do you have any recommendations?

Comment (pgoyal01) [reply doc/ref_impl/cntt-ri2/chapters/chapter04.md]: @rihabbanday You should open them within the [Anuket wiki RI2 WS page](https://wiki.anuket.io/display/HOME/RI2+-+Kubernetes+Reference+Implementation). Please see examples of pages under the [RM WS page](https://wiki.anuket.io/display/HOME/RM).

Comment (rihabbanday) [reply doc/ref_impl/cntt-ri2/chapters/chapter04.md]: @pgoyal01 Thanks for the info. I've noticed that these inconsistencies exist in other WSs as well (at least both RI's, both RC's), and I guess we need to agree on a common way of addressing (RA2/RI2/RC2 or RA-2/RI-2/RC-2) across all documents - I'll try to create a "Guidelines" page under "Anuket Common" as you suggested.

**Review by jgu17 — APPROVED** [review]

**Pull request merged**